docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -63,7 +72,7 @@ Administrators can abort the rotation and revert all changes any time before
 the rotation is completed by entering the `rollback` phase.
 
 ```sh
-$ tctl auth rotate --phase=rollback --manual
+$ tctl auth rotate --phase=rollback --type={host|user|db|jwt|all} --manual
```

Comment:
You're missing `openssh` added in Teleport 12 https://github.com/gravitational/teleport/pull/20020

Reply:
Good catch. I updated the docs.